api/src/main/java/jakarta/data/Sort.java

```diff
@@ -83,24 +83,28 @@
  * sort criteria.</p>
  *
  *
- * @param <T>         entity class of the property upon which to sort.
- * @param property    name of the property to order by.
- * @param isAscending whether ordering for this property is ascending (true) or descending (false).
+ * @param <T>         entity class of the entity attribute upon which to sort.
+ * @param property    name of the entity attribute to order by.
+ * @param isAscending whether ordering for this attribute is ascending (true) or descending (false).
  * @param ignoreCase  whether or not to request case insensitive ordering from a database with case sensitive collation.
  */
 public record Sort<T>(String property, boolean isAscending, boolean ignoreCase) {
 
     /**
-     * <p>Defines sort criteria for an entity property. For more descriptive code, use:</p>
+     * <p>Defines sort criteria for an entity attribute. For more descriptive code, use:</p>
      * <ul>
-     * <li>{@link #asc(String) Sort.asc(propertyName)} for ascending sort on a property.</li>
-     * <li>{@link #ascIgnoreCase(String) Sort.ascIgnoreCase(propertyName)} for case insensitive ascending sort on a property.</li>
-     * <li>{@link #desc(String) Sort.desc(propertyName)} for descending sort on a property.</li>
-     * <li>{@link #descIgnoreCase(String) Sort.descIgnoreCase(propertyName)} for case insensitive descending sort on a property.</li>
+     * <li>{@link #asc(String) Sort.asc(attributeName)}
+     *     for ascending sort on an entity attribute.</li>
+     * <li>{@link #ascIgnoreCase(String) Sort.ascIgnoreCase(attributeName)}
+     *     for case insensitive ascending sort on an entity attribute.</li>
+     * <li>{@link #desc(String) Sort.desc(attributeName)}
+     *     for descending sort on an entity attribute.</li>
+     * <li>{@link #descIgnoreCase(String) Sort.descIgnoreCase(attributeName)}
+     *      for case insensitive descending sort on an entity attribute.</li>
      * </ul>
      *
-     * @param property    name of the property to order by.
-     * @param isAscending whether ordering for this property is ascending (true) or descending (false).
+     * @param property    name of the entity attribute to order by.
```

Comment:
```suggestion
     * @param property name of the entity attribute to order by.
```
I would remove this space:

Reply:
Nope, ignore this.